sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -428,11 +428,25 @@ public void setStatus(Status status) {
 
   @Override
   public void recordException(Throwable exception) {
+    recordException(exception, null);
+  }
+
+  @Override
+  public void recordException(Throwable exception, Attributes additionalAttributes) {
     if (exception == null) {
       return;
     }
     long timestamp = clock.now();
-    Attributes.Builder attributes = Attributes.newBuilder();
+    final Attributes.Builder attributes = Attributes.newBuilder();
```

Comment:
why not just assign this to `additionalAttributes.newBuilder()` if the additional attributes are provided? I think the code will read more simply and be slightly more efficient.

Reply:
👍 Thanks. forgot this. : )